x-pack/plugins/security_solution/public/common/components/exceptions/use_fetch_or_create_rule_exception_list.test.tsx

```diff
@@ -207,6 +225,24 @@ describe('useFetchOrCreateRuleExceptionList', () => {
         expect(result.current[1]).toEqual(detectionExceptionList);
       });
     });
+    it('invokes onSuccess indicating that the rule did not change', async () => {
+      const onSuccess = jest.fn();
+      await act(async () => {
+        const { waitForNextUpdate } = renderHook(() =>
```

Comment:
Similar comment as above. You could also add `onSuccess` to the `render` function here https://github.com/elastic/kibana/blob/1dfc3654c838730c89ac386d673d7b472b5279fe/x-pack/plugins/security_solution/public/common/components/exceptions/use_fetch_or_create_rule_exception_list.test.tsx#L96, which kind of abstracts out the `renderHook` call.